tests/framework/extensions/authconfigs/openldap/openldapauth_test.go

```diff
@@ -0,0 +1,317 @@
+package openldapauth
```

Comment:
Tests don't go in extensions. It's either in validation or integration. But this one in particular should go into validation. It's also missing tags.